setup.py

```diff
@@ -348,7 +348,13 @@ def main():
         "linux": ["distro"],
         "proctitle": ["setproctitle"],
         "zipapp": ['importlib_resources; python_version < "3.7"'],
-        "full": ["ptk", "pygments", "distro"],
+        "full": [
+            "prompt-toolkit>=3",
+            "pyperclip",
+            "pygments>=2.2",
+            "distro; platform_system=='Linux'",  # PEP 508 platform specifiers
+            "setproctitle; platform_system=='Windows'",
```

Comment:
should we also add `gnureadline` here for OSX?

Reply:
Yes but that would make ptk kind of obsolete right ?

Reply:
not obsolete -- it would still be the default option, but some folks like to run with the `readline` backend (it can be helpful, especially in dumb shells, like in emacs)

Reply:
OK sure I will update now.

Reply:
done.